.github/files/build-reminder-comment/de-fusioned-files.js

```diff
@@ -7,6 +7,8 @@
  * Keep this list in alphabetical order.
  */
 const deFusionedFiles = [
+	'projects/plugins/jetpack/functions.opengraph.php',
+	'projects/plugins/jetpack/enhanced-open-graph.php',
```

Comment:
Should we keep these in alphabetical order?

Reply:
I went in and changed it anyway :)